### PR TITLE
fix(cli): start webdriver correctly

### PR DESCRIPTION
**Pull request by michael-siek**
See: #68

test: #72 

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [ ] Code is reviewed for security

### CODE REVIEW

File: packages/cli/lib/axe-test-urls.js [hunk 1]
```diff
@@ -3,15 +3,13 @@
 const WebDriver = require('selenium-webdriver');
 const AxeBuilder = require('@axe-core/webdriverjs');
 
-function testPages(urls, config, events) {
+async function testPages(urls, config, events) {
```
Comment (stephenmathieson) [inline packages/cli/lib/axe-test-urls.js:6]: Why is this an `async` function?

Comment (michael-siek) [reply packages/cli/lib/axe-test-urls.js:6]: I forgot to remove it when testing

Comment (michael-siek) [reply packages/cli/lib/axe-test-urls.js:6]: https://www.selenium.dev/selenium/docs/api/javascript/module/selenium-webdriver/index_exports_ThenableWebDriver.html

Comment (straker) [reply packages/cli/lib/axe-test-urls.js:6]: Michael and I paired on it. We tried to use the thenable interface as is without `await`ing it, and the unit tests for `startDriver` were causing us so many problems. We spent an hour trying to solve it and just gave up and made it work.

**Pull request merged**